src/sidebar/components/annotation-header.js

```diff
@@ -34,13 +37,26 @@ function AnnotationHeader({
             {replyCount} {replyPluralized}
           </a>
         </div>
-        {!isEditing && annotation.updated && (
+        {!isEditing && annotation.created && (
           <div className="annotation-header__timestamp">
-            <Timestamp
-              className="annotation-header__timestamp-link"
-              href={annotationLink}
-              timestamp={annotation.updated}
-            />
+            {hasBeenEdited && (
+              <span className="annotation-header__timestamp-edited">
+                (edited{' '}
+                <Timestamp
+                  className="annotation-header__timestamp-edited-link"
+                  href={annotationLink}
+                  timestamp={annotation.updated}
+                />
+                ){' '}
```

Comment:
I think it would be good to somehow space apart the "(edited ...)" and created timestamps a little more.